README.md

```diff
@@ -222,14 +211,87 @@ import './styles.scss'
 }
 ```
 
-## Development
```

Comment:
Why development section removed?

Reply:
Because it not longer applies, that test page was removed in favor of automated E2E.
If you like I can re-create the playground.

Reply:
That would be nice or mentioning how to locally test (using e2e) if it is prefered